src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
 
-function Dashboard() {
-  return <h2>Dashboard Two!</h2>;
-}
+import ApplyForBenefits from './apply-for-benefits/ApplyForBenefits';
+import ClaimsAndAppeals from './claims-and-appeals/ClaimsAndAppeals';
+import HealthCare from './health-care/HealthCare';
+
+const Dashboard = () => {
+  return (
+    <div className="vads-l-grid-container vads-u-padding-x--0">
```

Comment:
remind me where the `My VA` h1 comes from 😆

Reply:
I've added the h1 in [this draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15806) which I will open up for review after I merge this one 😄

Reply:
That works. Thanks.